### PR TITLE
Add location popup

### PR DESCRIPTION
**Pull request by tymoorej**
Select a location from a map to set. Also defaults map to the user's current position.

### CODE REVIEW

File: app/src/main/java/com/stl/skipthelibrary/BookDatabaseHelper.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+public class BookDatabaseHelper {
+    private static final String TAG = "BookDatabaseHelper";
+    private static final String BASE_URL = "https://www.googleapis.com/books/v1/volumes?";
+    private static final String QUERY_PARAM = "q";
+    private static final String QUERY_BASE = "=isbn:";
+    private static final String PRINT_TYPE = "printType";
+    private static final String PRINT_VALUE = "books";
+    private static final String METHOD = "GET";
+
+    public BookDatabaseHelper() {}
```
Comment (gillianpierce) [inline app/src/main/java/com/stl/skipthelibrary/BookDatabaseHelper.java]: Perhaps we should rename the DatabaseHelper class to UserDatabaseHelper for continuity?

File: app/src/main/java/com/stl/skipthelibrary/BorrowersBooksActivity.java [hunk 2]
```diff
@@ -30,25 +33,35 @@ protected void onCreate(Bundle savedInstanceState) {
...
-        books.add(new Book(new BookDescription("test Title", "test Sysnopsis",
+        books.add(new Book("test ISBN", new BookDescription("test Title", "test Sysnopsis",
                 "test author", new Rating()),"testUsername",
                 new RequestHandler(new State(BookStatus.REQUESTED,null, null)), null, new Rating()));
     }
 
     private void initRecyclerView(){
-        BookRecyclerAdapter adapter = new BookRecyclerAdapter(this, books);
+
+        Query songQuery = FirebaseDatabase.getInstance()
```
Comment (gillianpierce) [inline app/src/main/java/com/stl/skipthelibrary/BorrowersBooksActivity.java]: Why is this called songQuery? 
```suggestion
        Query bookQuery = FirebaseDatabase.getInstance()
```

File: app/src/main/java/com/stl/skipthelibrary/DatabaseHelper.java [hunk 3]
```diff
@@ -115,6 +123,20 @@ public void signOut(){
         firebaseAuth.signOut();
     }
 
+    /////////
+    // Book Functions
+    //TODO:Make this work
```
Comment (gillianpierce) [inline app/src/main/java/com/stl/skipthelibrary/DatabaseHelper.java]:
```suggestion
```

**Comment by LukeSlev** [comment]
@gillianpierce you may wanna look this over again, most of what you were looking at was not from this PR. I merged master in so should be able to see the actual diff now.

**Comment by gillianpierce** [comment]
> @gillianpierce you may wanna look this over again, most of what you were looking at was not from this PR. I merged master in so should be able to see the actual diff now.

Okey doke, thanks for letting me know!!

**Pull request merged**